server/apps/archive/archive_composite.py

```diff
@@ -134,8 +134,11 @@ def update_link(self, package_id, assoc, delete=False):
         if not delete:
             two_way_links.append({'package': package_id})
 
+        # do update without changing etag/last_modified
         updates = self.get_item_update_data(item, two_way_links, delete)
-        get_resource_service(endpoint).patch(item_id, updates)
+        updates['_etag'] = item['_etag']
```

Comment:
I understood that, but maybe a better solution would be to notify the client that the document updated so it can refetch the new document with the latest changes, including _etag...
My worry is that having an entity in mongo with the same _etag although they are not having the same content, it will break concurrency control...